common/timeline/timeline.go

```diff
@@ -6,6 +6,30 @@ import (
 	"github.com/dejaq/prototype/common/protocol"
 )
 
+type DeleteCaller uint8
+
+const (
+	DeleteCaller_Consumer DeleteCaller = iota
+	DeleteCaller_Producer
+)
+
+type DeleteMessages struct {
+	// Reference timestamp for calculate active leases
+	Timestamp uint64
+	// Who wants to delete messages
+	CallerType DeleteCaller
+	//Identity of who wants to delete
+	DeleterID  []byte
```

Comment:
I think CallerID is a more appropiate name?

Reply:
for sure, "CallerID is better"